### DISCUSSION TITLE
Custom Piano Music: Doki Doki Forever

### DISCUSSION THREAD

**Discussion by KarmaCreations** [Monika-After-Story/MonikaModDev#7791] [general - piano notes]
Hi! I've wanted to do this song for MAS for a while and I finally got around to doing it. Thank you to @multimokia for helping me find pnm.txt and fix some issues from there.
[dokidokiforever.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3899674/dokidokiforever.txt)
Updated expressions: [dokidokiforever.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3942849/dokidokiforever.txt)
Make sure you convert this to a .json before putting it in the folder :)
Here are the notes (sorry that it's so long):
P 9 Y U U Y U Y U Y U 9 R
E R Y P 9 U Y U Y U Y U 9 U Y
E R Y P 9 Y U U Y U Y U Y U 9 R
E E R Y P 9
Y U Y U Y U 9 U Y
Y Y Y 9 U Y R R R
E R Y Y Y
Y Y Y U 9 O 9 U U
[ P 9 U Y
R Y U Y [ P 9 9 P U
R R Y U Y [ P 9 U Y
R R Y 9 U Y 9 U Y
9 U Y R E R Y Y
[ P 9 U Y
R R Y U Y [ P 9 P U
R R Y U Y [ P 9 U Y
R Y 9 U Y 9 U Y
9 U Y R E R Y Y
9 U Y R E R Y Y
P 9 Y U U Y U Y U Y U 9 R
E E R Y P 9 U Y U U U 9 U Y
E E R Y
P 9 Y U U Y U U Y U 9 R
E E R Y P 9 Y U Y U Y U 9 U Y
Y Y Y 9 U Y R R R
E R Y Y Y
Y Y Y U 9 O 9 U U
[ P 9 U Y
R Y U Y [ P 9 P U
R R Y U Y [ P 9 U Y
R Y 9 U Y 9 U Y
9 U Y R E R Y Y
[ P 9 U Y
R R Y U Y [ P 9 P U
R R Y U Y [ P 9 U Y
R R Y 9 U Y 9 U Y
9 U Y R E R Y Y
Y Y Y 0 U Y R R R
E R Y Y Y
Y Y Y U 9 O 9 U U
R Y 9 U Y 9 U Y
9 U Y R E R Y Y
Y Y 9 U Y Y 9 U Y Y 9 U Y R E R Y Y
Enjoy!! (#3765)

**Comment by EmmaJK** [comment]
I wanted this so badly! Thank you

**Comment by EmmaJK** [comment]
It is perfect!!! Thank you again x
